lib/daemon/index.js

```diff
@@ -1,22 +1,56 @@
 // Command manager for daemonization
 
+var chalk = require('chalk');
+var cluster = require('cluster');
 var fs = require('fs');
+var net = require('net');
 var path = require('path');
-var cluster = require('cluster');
+var rimraf = require('rimraf');
+var xpipe = require('xpipe');
+
 var interrupt = require('./interrupt');
-var chalk = require('chalk');
 var mage = require('lib/mage');
 
 
 // Daemon configuration
 
 mage.core.config.setTopLevelDefault('daemon', path.join(__dirname, 'config.yaml'));
 
-var APP_LOCK_FILE = mage.core.config.get(['daemon', 'pidfile']);
-var CMD_LOCK_FILE = './.commandpidfile';
+const APP_LOCK_FILE = mage.core.config.get(['daemon', 'pidfile']);
+const CMD_LOCK_FILE = './.commandpidfile';
+const STDIO_PIPE_PATH = xpipe.eq('./.commandstdio');
 
 var timeouts = mage.core.config.get(['daemon', 'timeouts']) || {};
 
+function disableDaemonizedStdio(commandPid) {
+	if (!commandPid) {
+		return;
+	}
+
+	// Send stdio to a black hole
+	function disable(io) {
+		io.write = () => true;
```

Comment:
The previous code was `() => false;`.
What changed about that?

Reply:
`write` returns `true` in real life. I was fearing that `false` might somehow cause issues with dependencies.

Reply:
Because of exit logs?

Reply:
No, just because 'false' *may* be considered as an error. At any rate, returning `true` is more consistent.

Reply:
According to the Node.js doc:

```
Returns: <boolean> false if the stream wishes for the calling code to wait for the 'drain' 
event to be emitted before continuing to write additional data; otherwise true.
```

Since we are *never* going to call drain, true is safer.

Reply:
Ah yes, indeed 👍 
But friendly cc to @christopherobin who was involved in the original `return false` I think?

Reply:
I was the one who wrote that for him, he just picked up the code from a separate branch I had created to solve some issues he was having with his initial PR.